main-actions/src/main/scala/sbt/Package.scala

```diff
@@ -152,7 +162,17 @@ object Package {
       homepage map (h => (IMPLEMENTATION_URL, h.toString))
     }: _*)
   }
-  def makeJar(sources: Seq[(File, String)], jar: File, manifest: Manifest, log: Logger): Unit = {
+  @deprecated("Please specify whether to use a static timestamp", "1.3.4")
```

Comment:
I think we should do this as part of 1.4.0.

Reply:
:+1: